src/components/shared/Promotional/Promotional.test.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import DopplerIntlProvider from '../../../i18n/DopplerIntlProvider';
+import { Promotional } from './Promotional';
+
+describe('Promotional component', () => {
+  const texts = {
+    title: 'a title',
+    description: 'some description',
+    strongText: 'an strong text',
+    actionText: 'PLUS PLANS',
+  };
+
+  const urls = {
+    actionUrl: '/path/to/plus/plans',
+    iconUrl: 'icon.svg',
+    imageUrl: 'image.jpg',
+  };
+
+  it('should render component without features', () => {
```

Comment:
@ramniel  just one thing. Missing add:
`expect(screen.getByRole('list')).not.toBeInTheDocument();`

I really liked the tests, very easy to read.

Reply:
Thanks!

Reply:
Advice: Use the `query* `variants for asserting that an element cannot be found `expect(screen.queryByRole('list')).not.toBeInTheDocument();`